dim_checks/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/dim_checks.yaml

```diff
@@ -0,0 +1,88 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
```

Comment:
Same [comment in this PR43](https://github.com/mozilla/dim/pull/43/files#r1024283589), we can make this validation more meaningful to identify peaks in data right? Could we please use the a value higher or equal than the average or minimum value per period?
I'm curious about @gkabbz thoughts, since there have been some incidents of data peaks and the monitoring system could notify early about, but we also have the overwatch system.